pkg/profilecreator/profilecreator_test.go

```diff
@@ -188,6 +188,26 @@ var _ = Describe("PerformanceProfileCreator: Populating Reserved and Isolated CP
 			_, _, err := handle.GetReservedAndIsolatedCPUs(reservedCPUCount, splitReservedCPUsAcrossNUMA)
 			Expect(err).To(HaveOccurred())
 		})
+		It("Errors out in case splitReservedCPUsAcrossNUMA is enabled and number of reserved CPUs per number of NUMA nodes are odd", func() {
```

Comment:
If HT is enabled, right?

Reply:
Yep! Sorry that was a mistake, will fix.